packages/notifier/test/test-notifier.js

```diff
@@ -1,70 +1,99 @@
+// @ts-check
 // eslint-disable-next-line import/no-extraneous-dependencies
 import '@agoric/install-ses';
 import { test } from 'tape-promise/tape';
 import { produceNotifier } from '../src/notifier';
 
-test('notifier - initital state', t => {
+/**
+ * @template T
+ * @typedef {import('../src/notifier').NotifierRecord<T>} NotifierRecord<T>
+ */
+
+test('notifier - initial state', async t => {
+  /** @type {NotifierRecord<1>} */
```

Comment:
why not `NotifierRecord<number>`?

Reply:
Yes, `number` is more general, but it's fine to say that only the number `1` is the type of the state.